### ISSUE TITLE
Displaying of command errors could probably use some tweaking

### ISSUE THREAD

**Issue by autarch** [justjanne/powerline-go#11]
For some commands a non-zero exit isn't really indicative of an error. As an example, run `git grep frobnitz` in this repo. This returns a non-zero exit code because nothing was found, but showing this as an error had me quite confused for a bit.

I have no idea how to handle this. Maybe make it possible to ignore certain error codes via config?

**Comment by justjanne** [comment]
The problem is that there is a standard (error codes between 64 and 127 are errors of the user or program, 128 means the error code is invalid, 129 to 255 are errors resulting from signals being sent), but this standard isn't really used.

Every program uses another standard for error codes, and generally 1 is used as generic error, 2 for usage, etc.

Grep and git grep use them differently, though.

I'm not sure if there is a better way, but I wanted to use meaningful names instead of numbers whenever possible.

**Comment by autarch** [comment]
Yeah, I'm really not sure how to do anything better. Maybe this just needs some documentation updates.

**Comment by mpepping** [comment]
In this context in will be an improvement if the showing of meaningful names for errors, or the 'plain' exit codes, can be configured. In the case of the `grep` and `git grep` examples the parsing of the return prompt (as a human 😎) would be easier.

**Comment by mpepping** [comment]
Added a PR to make either meaningful name (default) or numeric exit codes configurable.
